docs/earthly-command/earthly-command.md

```diff
@@ -646,6 +646,148 @@ List secrets the current account has access to.
 
 Removes a secret from the secrets store.
 
+## earthly registry
+
+Contains sub-commands for managing registry access in cloud-based secrets.
+
+#### earthly registry setup
+
+#### Synopsis
+
+* ```
+  earthly registry setup [--org <org> --project <project>] [--cred-helper <none|ecr-login|gcloud>] ...
+  ```
+
+###### username/password based registry (`--cred-helper=none`)
+
+* ```
+  earthly registry setup --username <username> --password <password> [<host>]
+  earthly registry setup --org <org> --project <project> --username <username> --password <password> [<host>]
```

Comment:
Many of our other commands have org and project flags one level up, so that they can be shared between the various sub-commands.

So for example

```
earthly registry --org <org> --project <project> setup --username <username> --password <password> [<host>]
```

Perhaps we should do the same, for consistency.

Reply:
good idea.

Reply:
I'll change other docs / FAQ docs on my end too.